contracts/BinaryOptionMarket.sol

```diff
@@ -49,99 +42,77 @@ contract BinaryOptionMarket is Owned, MixinResolver, IBinaryOptionMarket {
     /* ========== STATE VARIABLES ========== */
 
     Options public options;
-    Prices public prices;
     Times public times;
     OracleDetails public oracleDetails;
     BinaryOptionMarketManager.Fees public fees;
-    BinaryOptionMarketManager.CreatorLimits public creatorLimits;
```

Comment:
There is no longer a concept of an skew, so minimal skew is gone as well. Minimal requirement of 1000 sUSD to create a market is kept to ensure some initial orderbook liquidity.